src/javascripts/components/MessageLog.vue

```diff
@@ -1,13 +1,20 @@
 <template>
-  <ul>
-    <h1> <u>チャットリスト</u> </h1>
+  <ul class="container">
     <template v-for="message in messages">
-      <li :key="message.key">
-
-        <h3>{{message.name}}</h3>
-        <p>{{message.text}}</p>
-        <h5>{{message.time}}</h5>
-      </li>
+      <v-card
+        class="textbox"
+        v-bind:class="{yourMessage: message.userId === userId}"
```

Comment:
v-bindの記法は `:foo` に統一しちゃっていいと思います！
```suggestion
        :class="{yourMessage: message.userId === userId}"
```